### ISSUE TITLE
Fasttext low test accuracy on small dataset

### ISSUE THREAD

**Issue by Nil0801** [facebookresearch/fastText#1178]
Hello,
First of all thank you for such an amazing library. I am using this library to predict sentiments of the sentence by categorizing  training data into five different labels. I am training data on 5000 rows of data. I tried training using different hyperparameters but still not getting desired result. **My Training accuracy is around 90% but Testing is only 45%. Is there anyway I can improve that?** Also when I try giving simple text for prediction like model.predict('fantastic') or model.predict('Fantastic') it gives different result for same text with upper and lower case. It gives me Negative for first one and Neutral for second one even though it is classified as Positive. **Is there anyway to improve this?** 

Thank you so much in advance

**Comment by ajaech** [comment]
If you don't want the model to be allowed to give different predictions for upper and lower case words you might consider lower casing all of the text before training the fasttext model.
